pavlov_vr/egg-pavlov-v-r.json

```diff
@@ -4,13 +4,13 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2022-01-09T09:30:49-06:00",
+    "exported_at": "2022-01-09T09:55:08-06:00",
     "name": "Pavlov VR",
     "author": "iamkubi@gmail.com",
     "description": "Pavlov is a multiplayer shooter in VR",
     "features": null,
     "images": [
-        "quay.io\/pterodactyl\/core:source"
+        "ghcr.io\/smooklu\/games:source"
```

Comment:
What's better about it?

Reply:
The current `core:source` image is missing the dependency `libc++-dev` so I created my own image with that dependency. I also added a extra variable  so anyone can install Pavlov Beta, Shack or Shack RC versions.

Reply:
Without that dependency, Pavlov Dedicated Server refuses to run. ```/home
/container/Pavlov/Binaries/Linux/PavlovServer: error while loading shared libraries: libc++.so.1: cannot open shared object file: No such file or directory```